test/integration/cloudresource_e2e_test.go

```diff
@@ -545,7 +544,7 @@ var _ = Describe(fmt.Sprintf("%s,%s: NetworkPolicy On Cloud Resources", focusAws
 			[]string{apachePort}, false)
 		verifyIngress(kind, ids[appliedIdx], ips[appliedIdx], srcVMs, oks, false)
 
-		By(fmt.Sprintf("Ingress NetworkPolicy on %v by name label selector", kind))
+		logf.Log.Info("Ingress NetworkPolicy on %v by name label selector", kind)
```

Comment:
This is giving an error. 
Should be changed to fmt.Sprintf or change the log message to key and value pair.
  2023-04-12T07:45:50.478Z	DPANIC	integration/cloudresource_e2e_test.go:547	odd number of arguments passed as key-value pairs for logging	{"ignored key": "VirtualMachine"}